VKApp/VKApp/Services/PhotoCacheService.swift

```diff
@@ -0,0 +1,130 @@
+// PhotoCacheService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+private protocol DataReloadable {
+    func reloadRow(atIndexpath indexPath: IndexPath)
+}
+
+/// Кеширование фото
+final class PhotoCacheService {
+    // MARK: - Private properties
+
+    private let cacheLifeTime: TimeInterval = 30 * 24 * 60 * 60
+    private static let pathName: String = {
+        let pathName = Constants.pathName
+        guard let cachesDirectory = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first
+        else { return pathName }
+        let url = cachesDirectory.appendingPathComponent(pathName, isDirectory: true)
+        if !FileManager.default.fileExists(atPath: url.path) {
+            try? FileManager.default.createDirectory(
+                at: url,
+                withIntermediateDirectories: true,
+                attributes: nil
+            )
+        }
+        return pathName
+    }()
+
+    private let container: DataReloadable
+    private let networkService = NetworkService()
+
+    private var images = [String: UIImage]()
```

Comment:
```suggestion
    private var images: [String: UIImage] = [:]
```
из гайда
<img width="859" alt="Снимок экрана 2022-12-12 в 01 11 34" src="https://user-images.githubusercontent.com/86570514/206931815-199ca6af-a62c-41ac-a6af-657d7daeeba5.png">